sklearn/preprocessing/data.py

```diff
@@ -1263,6 +1263,7 @@ class PolynomialFeatures(BaseEstimator, TransformerMixin):
     See :ref:`examples/linear_model/plot_polynomial_interpolation.py
     <sphx_glr_auto_examples_linear_model_plot_polynomial_interpolation.py>`
     """
+
```

Comment:
Please do not change unrelated things. It makes your contribution harder to review and may introduce merge conflicts to other pull requests.

Please remove the inserted blank lines

Reply:
these lines were introduced by AutoPep8, will skip that from now

Reply:
I don't get why autopep8 does that. It is quite clearly not specified in PEP8 and against the conventions used in PEP257.

Reply:
I would like you to remove these blank lines. It is not our convention. And extra changes means extra merge conflicts